### DISCUSSION TITLE
How to install Mockoon on RHEL7?

### DISCUSSION THREAD

**Discussion by l3Lite** [mockoon/mockoon#823] [Help]
I want to install Mockoon on RHEL7 by using rpm file that you provided in [this](https://github.com/mockoon/mockoon/releases/download/v1.20.0/mockoon-1.20.0.rpm).

I installed Mockoon by using `yum install mockoon-1.20.0.rpm` .
And when I tried to start my mock APIs `mockoon start --no-sandbox --data demo.json` 

I got below error
![image](https://user-images.githubusercontent.com/29365546/193258004-9041463c-72cb-425f-9246-8c7a232a1ce7.png)

Another way that I've tried is installing it via `npm install -g mockoon-1.20.0.tar.gz` which I downloaded from [this](https://github.com/mockoon/mockoon/archive/refs/tags/v1.20.0.tar.gz)
And it didn't work,
`-bash: mockoon-cli: command not found`

My server is inside the private network. So I cannot install Mockoon via `snap install mockoon`.

Are there any other ways to install Mockoon on RHEL7?
Please suggest me.

Thank you.

**Comment by 255kb** [comment]
Mockoon is primarily a desktop application. The binaries you see on the download page or GitHub releases are for the desktop application.
What you are looking for is the CLI companion which is a Node.js package that you have to install with NPM: `npm i -g @mockoon/cli`
There is no binary for the CLI.

I suggest that you follow the getting started tutorials so you get an idea what are the differences between the two applications:
https://mockoon.com/tutorials/getting-started/
https://mockoon.com/tutorials/run-mock-api-anywhere-cli/

**Reply by l3Lite** [reply]
I've already tested this way but my server is inside private network. So I cannot run `npm i -g @mockoon/cli`.
Do you have another options?

**Reply by 255kb** [reply]
CLI is currently not distributed in another format. 
Maybe you could try to change NPM global directory: https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally
